modules/Agent.py

```diff
@@ -415,6 +417,10 @@ def __online(self,newMethod=True):
                     episodes+=1
```

Comment:
we are incrementing the episode, as well as iteration, which doesn't seem right. May we can remove either of the two unless needed.